crates/ruff/src/rules/flake8_simplify/rules/ast_bool_op.rs

```diff
@@ -503,30 +504,53 @@ pub fn expr_or_not_expr(checker: &mut Checker, expr: &Expr) {
     }
 }
 
-pub fn is_short_circuit(
-    ctx: &Context,
-    expr: &Expr,
-    expected_op: &Boolop,
-) -> Option<(Location, Location)> {
+fn get_bool(expr: &Expr) -> Option<bool> {
+    if let ExprKind::Constant {
+        value: Constant::Bool(value),
+        ..
+    } = expr.node
+    {
+        Some(value)
+    } else {
+        None
+    }
+}
+
+pub fn is_short_circuit(ctx: &Context, expr: &Expr, expected_op: &Boolop) -> Option<Edit> {
     let ExprKind::BoolOp { op, values, } = &expr.node else {
         return None;
     };
     if op != expected_op {
         return None;
     }
-    let short_circuit_value = match op {
-        Boolop::And => false,
-        Boolop::Or => true,
+    let (short_circuit_value, short_circuit_string) = match op {
+        Boolop::And => (false, "False".to_string()),
+        Boolop::Or => (true, "True".to_string()),
     };
```

Comment:
Nit: Can we move the `to_string` call to where we create the `Edit` to avoid allocating a string on the heap if there are no diagnostics.